pages/index.vue

```diff
@@ -1,4 +1,57 @@
+<script lang="ts" setup>
+// Mock data, to be replaced with real data from the API
+const statuses = ['todo', 'doing', 'done']
+
+const issues = [
+  {
+    id: '42ddc58de82757cf729f6c1b4fba0536',
+    title: 'Implement User Authentication',
+    status: 'todo',
+  },
+  {
+    id: '703abf5af2fc9c4a3e7dbb1193e83511',
+    title: 'Design Database Schema',
+    status: 'todo',
+  },
+  {
+    id: 'a0a85d0c34f73ec3b8c213409b2e8d29',
+    title: 'An issue with a really long name to see whether the ui handles it well or not',
+    status: 'todo',
+  },
+  {
+    id: '26ec420ba69483f7298f3842b8e759f4',
+    title: 'Create API Endpoints',
+    status: 'doing',
+  },
+  {
+    id: '41922e1ed28e9eaf7b6124af8f42bdfa',
+    title: 'Fix Login Bug',
+    status: 'doing',
+  },
+  {
+    id: 'dfbbf053f6ecbecd66f966574f6bd6ed',
+    title: 'Improve Page Load Performance',
+    status: 'done',
+  },
+  {
+    id: '81a2a5b64efdec72e59c5de36b7dc997',
+    title: 'Add Unit Tests',
+    status: 'done',
+  },
+]
+</script>
+
 <template>
-  <TestComponent />
-  <NuxtWelcome />
-</template>
\ No newline at end of file
+  <main class="flex min-h-screen flex-col">
```

Comment:
Well done thinking about `<main>`! :clap: 

But it's odd having it only on the index page. Nuxt has something called a ["default Layout"](https://nuxt.com/docs/guide/directory-structure/layouts#default-layout). We should move it there instead. :)

Reply:
Great pointer, added!